src/services/objectives.js

```diff
@@ -185,6 +186,17 @@ export async function getObjectivesByReportId(reportId) {
           activityReportId: reportId,
         },
         required: true,
+        include: [
+          {
+            model: ActivityReport,
+            as: 'activityReport',
+            required: true,
+            attributes: ['id', 'activityRecipientType'],
+            where: {
+              activityRecipientType: 'other-entity',
+            },
+          },
+        ],
```

Comment:
This is not needed if you just want to get the otherEntity Objectives. Just check that the grantId is null and otherEntityId is not null on the objective. As the activityRecipientType on the report is likely to be removed soon.

Reply:
Good idea! I will update this 😄

Reply:
This has been updated.